include/pando-lib-galois/graphs/dist_local_csr.hpp

```diff
@@ -1136,7 +1136,8 @@ class DistLocalCSR {
    */
   std::uint64_t getVertexLocalIndex(VertexTopologyID vertex) {
     std::uint64_t hostNum = static_cast<std::uint64_t>(galois::localityOf(vertex).node.id);
-    return fmap(arrayOfCSRs[hostNum], getVertexIndex, vertex);
+    HostIndexedMap<CSR> arrayOfCSR = arrayOfCSRs[hostNum];
+    return fmap(arrayOfCSR[hostNum], getVertexIndex, vertex);
```

Comment:
This seems to be the wrong fix, you need to call getCSR(hostNum);

Reply:
Do you mean that
```
HostIndexedMap<CSR> arrayOfCSR = arrayOfCSRs.getCSR(hostNum);
return fmap(..);
```
?
I will check what the difference of [] and getCSR are

Reply:
nvm. i got what you meant.

Reply:
(even though it seems like it is not wrong but not well exploited api?)